documentation/src/main/asciidoc/server_guide/server_guide.adoc

```diff
@@ -607,6 +607,45 @@ The REST connector differs from the above connectors because it piggybacks on th
 
 ----
 
+==== Multi-tenant router
+The Multi-tenant router endpoint facades one or more REST or Hot Rod connectors. Its main purpose is to forward client requests into proper container.
+
+In order to properly configure the routing, `socket-binding` attributes of other connectors must be disabled and additional attribute `name` must be used as shown below:
+
```

Comment:
That is correct. You may however create another `<hotrod-endpoint>` in the configuration and expose the same Cache Container.